server/src/server/api/middleware/responseLoggerMiddleware.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response, NextFunction } from "express";
+
+export const ResponseLogger = (
+  request: Request,
+  response: Response,
+  next: NextFunction
+) => {
+
+  // 4/2/2020, 10:39:06 AM: [1347a8a782381] Response GET 200 - /
+  console.info(`${new Date().toLocaleString()}: [${(request as any).id}] Response ${request.method} ${response.statusCode} - ${request.path}`);
+  
```

Comment:
Lets output this in json. and use a UTC timestamp

Reply:
where is the correlationId?

Reply:
`[${(request as any).id}]` is the correlationId

Reply:
Changed output to JSON and now using UTC - 0f5d7ca